pkg/webui/components/panel/index.js

```diff
@@ -38,6 +38,7 @@ const Panel = ({
   className,
   messageDecorators,
   divider,
+  target,
```

Comment:
It would be clearer to prefix all link related props with e.g. `shortcutLink`

`shortCutLinkTitle`
`shortCutLinkPath`
`shortCutLinkTarget`